tests/zfs-tests/tests/functional/checksum/setup.ksh

```diff
@@ -33,4 +33,5 @@
 
 verify_runnable "global"
 
-default_mirror_setup $DISKS
+truncate -s 1G $TEST_BASE_DIR/disk0 $TEST_BASE_DIR/disk1
+default_mirror_setup $TEST_BASE_DIR/disk0 $TEST_BASE_DIR/disk1
```

Comment:
Rather than change to using file based vdevs, could we instead add
`sysctl -n kern.geom.debugflags` to corrupt_blocks_at_level for freebsd?
That would make the function more generally useful.

Reply:
That works. I couldn't think of a good way to restore the flags if something fails inside the function, but things are probably already pretty screwed up if that happens anyway.